src/machine/machine_atsamd21.go

```diff
@@ -1157,7 +1157,7 @@ func (spi SPI) Configure(config SPIConfig) error {
 		dataOrder = 1
 	}
 
-	// Set SPI master
+	// Set SPI mode to chip
```

Comment:
Again, I believe the new name is "controller".

Reply:
Changed.